content/intro-to-storybook/react/en/composite-component.md

```diff
@@ -240,12 +247,11 @@ Create a test file called `src/components/TaskList.test.js`. Here, we’ll build
 import React from 'react';
 import ReactDOM from 'react-dom';
 import TaskList from './TaskList';
-import { withPinnedTasks } from './TaskList.stories';
+import { WithPinnedTasks } from './TaskList.stories';
```

Comment:
CSF FTW!

Reply:
the documenation is starting to get along real nice. 💯